conductr_cli/constants.py

```diff
@@ -2,6 +2,8 @@
 
 CONDUCTR_SCHEME = 'CONDUCTR_SCHEME'
 
+CONDUCTR_BUNDLE_PROXY_PORT = 9000  # The default proxy port for bundles.
```

Comment:
I think we would be able to make do without this constant. Refer to the comments below.